bxt-strafe/src/steps.rs

```diff
@@ -545,6 +557,47 @@ impl<S: Step> Step for ResetFields<S> {
             state.strafe_cycle_frame_count = 0;
         }
 
+        // If we have some acceleration, then this kicks in.
+        // It will preserve the final value across split segments.
+        if let Some(AutoMovement::Strafe(StrafeSettings {
+            type_:
+                StrafeType::MaxAccelYawOffset {
+                    start,
+                    target,
+                    accel,
+                },
+            dir,
+        })) = frame_bulk.auto_actions.movement
+        {
+            let right = matches!(dir, StrafeDir::Right);
+
+            // Flip start and target when accel is negative.
+            state.max_accel_yaw_offset_value = (state.max_accel_yaw_offset_value + accel)
+                .max(start)
+                .min(target);
```

Comment:
```suggestion
                .clamp(start, target);
```
This should work I think.

Reply:
This would crash because 

```
Message:  min > max, or either was NaN. min = -0.9614217, max = -1.9672347
```

Reply:
Ah, right